docs/_articles/en/apex/language-server.md

```diff
@@ -3,33 +3,30 @@ title: Apex Language Server
 
 The Apex Language Server is an implementation of the Language Server Protocol 3.0 [specification](https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md). The Language Server Protocol allows a tool (in this case, VS Code) to communicate with a language smartness provider (the server). We built the Apex Language Server using this common specification to enable our tooling partners to improve the smartness of their tools.
 
 For more information, watch the video of our Dreamforce ’17 presentation, [Building Powerful Tooling For All IDEs Through Language Servers](https://www.salesforce.com/video/1765282/).
 
-[![Dreamforce '17 Presentation](./images/apex-language-server-presentation-dreamforce-17.png)](https://www.salesforce.com/video/1765282/)
+[![Dreamforce '17 Presentation](../../../images/apex-language-server-presentation-dreamforce-17.png)](https://www.salesforce.com/video/1765282/)
 
-## Installation
-
-We recommend that you install the [Salesforce Extension Pack](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode) from the VS Code Marketplace. The extension pack includes the Apex Language Server and its integration with VS Code.
-
-## Usage
+## Integrate with the Apex Language Server
 
-See [Apex](./en/apex/writing).
+If you are a developer looking to integrate with the Apex Language Server, use the [apex-jorje-lsp.jar](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-apex/out/apex-jorje-lsp.jar) file.
 
-## Clear the Cache
+See the [languageServer.ts](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-apex/src/languageServer.ts) file for an example of initializing and communicating with the Apex Language Server.
 
-To clear the Apex Language Server's cache, delete the `PROJECT_DIR/.sfdx/tools/apex.db` file and restart VS Code.
+## Apex Language Server Status
 
-## Integrate with the Apex Language Server
+You can check the status of the Apex Language Server from the status bar. Click `{}` in the status bar and pin `Indexing complete ✓` and restart VS Code. Next time you open an Apex file, the status of the Apex Language Server is visible in the status bar.
 
-If you are a developer looking to integrate with the Apex Language Server, use the [apex-jorje-lsp.jar](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-apex/out/apex-jorje-lsp.jar) file.
+![Apex Language Server Status](../../../images/apexlspstatus.png)
 
-See the [languageServer.ts](https://github.com/forcedotcom/salesforcedx-vscode/blob/develop/packages/salesforcedx-vscode-apex/src/languageServer.ts) file for an example of initializing and communicating with the Apex Language Server.
+## Clear the Cache
 
-For more information, consult these resources:
+To clear the Apex Language Server's cache, delete the `PROJECT_DIR/.sfdx/tools/apex.db` file and restart VS Code.
```

Comment:
This change also introduced a couple of changes to the directory structure.

1. The apex.db directory correlates to the core release. The file can be found in `PROJECT_DIR/.sfdx/tools/NNN/apex.db`, where NNN = current release. (We are a bit ahead on this front, with NNN = 246).
2. A new folder has been added in `PROJECT_DIR/.sfdx/tools/NNN/` named `StandardApexLibrary`. The directory holds the faux classes for a given release's standard apex classes. These files for goto definition in the same way as the sobjects in `.apex/tools/sobjects`.
3. To "reindex everything" the customer should rm folder `.sfdx/tools/NNN`